libvast/src/system/source_command.cpp

```diff
@@ -78,37 +78,36 @@ caf::message source_command(const command::invocation& invocation,
   auto guard = system::signal_monitor::run_guarded(
     sig_mon_thread, sys, defaults::system::signal_monitoring_interval, self);
   // Get node components.
-  auto components
-    = get_node_component<accountant_atom, importer_atom>(self, node);
+  auto components = get_node_components(self, node, {"accountant", "importer"});
   if (!components)
     return make_message(components.error());
   if (auto& accountant = (*components)[0]) {
     VAST_DEBUG(invocation.full_name, "assigns accountant to source");
-    self->send(src, actor_cast<accountant_type>(*accountant));
+    self->send(src, actor_cast<accountant_type>(accountant));
   }
   // Connect source to importer.
   auto& importer = (*components)[1];
   if (!importer)
-    return make_message(importer.error());
+    return make_message("couldnt lookup IMPORTER");
```

Comment:
`caf::message` works a bit like `std::any`, so in this case we need to make sure it contains an error instead of a string.
```suggestion
    return make_message(make_error(ec::missing_component, "importer"));
);
```